tools/Get-AdditionalDataBin.ps1

```diff
@@ -0,0 +1,33 @@
+$ErrorActionPreference = "Stop"
+
+function DownloadFile($url, $outputPath) {
```

Comment:
Doesn't the `Get-CatalogBin.ps1` script have a similar method? Maybe combine the two as `Get-DataFiles.ps1` so there's no duplication

Reply:
Sounds good idea. I will merge this PR once we have the exception.bin file in the blob storage, then make a new PR to merge the duplicated code.